metrics/grafana/reparo.json

```diff
@@ -0,0 +1,609 @@
+{
```

Comment:
what the hell is this file? I can't find any related metrics, I think it's never used.

Reply:
cp from https://github.com/pingcap/tidb-ansible/blob/master/scripts/reparo.json

Reply:
seems is uselss, I will delete it

Reply:
how about also remove the ones about kafka since it's not related to this repo(like downstream tidb).

Reply:
ok